spec/controllers/idv/image_uploads_controller_spec.rb

```diff
@@ -511,6 +512,7 @@
               selfie_quality_good: true,
               birth_year: 1938,
               zip_code: '12345',
+              selfie_attempts: 1,
```

Comment:
If we don't actually process the selfie for a request, I don't know that I'd expect this to be counted as a selfie attempt?